spec/requests/v0/user_spec.rb

```diff
@@ -7,6 +7,15 @@
   include SchemaMatchers
   include SM::ClientHelpers
 
+  VCR.configure do |config|
+    config.register_request_matcher :wildcard_path do |request1, request2|
+      # Removes the user id and icn after `/isValidSMUser/` to handle any user id and icn
+      path1 = request1.uri.gsub(%r{/isValidSMUser/.*}, '/isValidSMUser')
+      path2 = request2.uri.gsub(%r{/isValidSMUser/.*}, '/isValidSMUser')
+      path1 == path2
+    end
+  end
+
```

Comment:
Configuring VCR in the middle of specs can lead to unintended consequences. If you need a custom matcher like this, you should define it in the [main VCR configuration](https://github.com/department-of-veterans-affairs/vets-api/blob/c37abd316d610b6230750bc60934e3577315bed2/spec/support/vcr.rb#L3) and give it a more descriptive name like `uri_ignoring_path_parameter`. Additionally, this helps eliminate repeated configuration across your specs

Reply:
Thanks for the idea! Wasn't sure if I should put it in the main config or not. Is the name `uri_ignoring_path_parameters` descriptive enough or should I go with something even more so like `uri_ignoring_path_parameters_after_isvalidsmuser`?